dropwizard-testing/src/main/java/io/dropwizard/testing/junit5/DAOTestExtension.java

```diff
@@ -70,6 +74,20 @@ public void after() {
         daoTest.after();
     }
 
+    @Override
+    public void beforeAll(ExtensionContext extensionContext) throws Exception {
+        try {
+            daoTest.before();
+        } catch (Throwable e) {
+            throw new HibernateException(e);
+        }
+    }
+
+    @Override
+    public void afterAll(ExtensionContext extensionContext) {
+        daoTest.after();
```

Comment:
Would it make sense to call `DAOTestExtension#after()` here so that the behavior is guaranteed to be the same?